packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/base/common_ui_components/app_open_mail_widget.dart

```diff
@@ -0,0 +1,26 @@
+{{> licence.dart }}
+
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:open_mail/open_mail.dart';
+import 'package:rx_bloc/rx_bloc.dart';
+
+typedef AppOpenMailWidgetCallback<BlocType extends RxBlocTypeBase>
+    = Stream<List<MailApp>> Function(BlocType bloc);
+
+class AppOpenMailWidget<BlocType extends RxBlocTypeBase>
+    extends StatelessWidget {
+  const AppOpenMailWidget({super.key, required this.openMailState});
+
+  final AppOpenMailWidgetCallback<BlocType> openMailState;
```

Comment:
Would be great if we head this documented.

Reply:
Done